tests/foreman/ui/test_activationkey.py

```diff
@@ -1227,3 +1227,39 @@ def test_positive_ak_with_custom_product_on_rhel6(session, rhel6_contenthost, ta
         ak = session.activationkey.read(ak.name, widget_names='content_hosts')
         assert len(ak['content_hosts']['table']) == 1
         assert ak['content_hosts']['table'][0]['Name'] == rhel6_contenthost.hostname
+
+
+@pytest.mark.rhel_ver_list([7, 8, 9])
+def test_positive_custom_products_disabled_by_default_ak(
+    session,
+    default_location,
+    setup_content,
+):
+    """Verify that repositories can be filtered by type and that repositories
+    are Disabled by default on activation keys
+
+    :id: 3e6793ff-3501-47f6-8cd6-e24a60bdcb73
+
+    :steps:
+        1. Create custom product and upload repository
+        2. Attach to activation key
+        3. Filter Repositories by "Custom" type
+        4. Assert that custom proudct is visible and disabled by default
+        5. Filter Repositories by "Red Hat" type
+        6. Assert custom product is not visible
+
+    :expectedresults: Custom products should be filtered and Disabled by default.
+    """
```

Comment:
Can you link any BZ or RFE which introduce this change?

Reply:
I dont have a specifc BZ for this one. But i can attach the epic BZ